apiserver/modelmanager/modelmanager.go

```diff
@@ -537,7 +536,14 @@ func (m *ModelManagerAPI) ModelInfo(args params.Entities) (params.ModelInfoResul
 	return results, nil
 }
 
+type pluggableAuthFunc func(names.UserTag) error
+
 func (m *ModelManagerAPI) getModelInfo(tag names.ModelTag) (params.ModelInfo, error) {
+	return m.getModelInfoPluggableAuth(tag, m.authCheck)
+}
+
+func (m *ModelManagerAPI) getModelInfoPluggableAuth(tag names.ModelTag,
```

Comment:
I think my point is that the current authCheck code should be sufficient without any changes needed